### PR TITLE
Generic incremental mode on destination

### PR DESCRIPTION
**Pull request by ChristopheDuong**
## What
Destination can now aggregate data over multiple runs

## How
DBT make copies of tables _raw produced by each destination and aggregates them in a _history_raw_out table

It also normalize tables and aggregates normalized tables in _history_out table

See an example of multiple runs over bigquery here:
https://console.cloud.google.com/bigquery?project=dataline-integration-testing&p=dataline-integration-testing&d=chris_data&t=exchange_rate_raw&page=table

We get the same data row for different _emitted_at and _normalized_at dates

It doesn't use any comparable key yet since it it not defined yet on the source side
and this incremental behavior should be possible to toggle on or off depending on the catalog but that is not done yet

And if schema/catalogs are changing then this is going to break down

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1200367912513076/1200368132640656) by [Unito](https://www.unito.io)

### CODE REVIEW

**Comment by ChristopheDuong** [comment]
I moved the Normalization Factory/Runner into the java base destination instead of being in the Worker module...

So instead of building a Docker Process that runs DBT, it's running a Bash Process that runs DBT to be part of the same docker image as the destination... let me know if that's what you expected?

File: airbyte-integrations/bases/base-java/Dockerfile [hunk 1]
```diff
@@ -1,5 +1,21 @@
 FROM openjdk:14.0.2-slim
 COPY --from=airbyte/integration-base:dev /airbyte /airbyte
+COPY --from=airbyte/normalization:dev /airbyte /airbyte
```
Comment (ChristopheDuong) [inline airbyte-integrations/bases/base-java/Dockerfile:3]: I didn't find in the gradle tasks how to make the base-java:airbyteDocker depends on airbyte/normalization:airbyteDocker though...

Comment (jrhizor) [reply airbyte-integrations/bases/base-java/Dockerfile:3]: You need something like `implementation files(project(':airbyte-integrations:bases:base-normalization').airbyteDocker.outputs)` in gradle

Comment (ChristopheDuong) [reply airbyte-integrations/bases/base-java/Dockerfile:3]: I've tried that but i really don't see why doesn't like it...
To me, there is no significant differences in the build.gradle between the other projects?

Screenshot 2020-11-19 at 21 37 08

**Comment by cgardens** [comment]
@ChristopheDuong okay if we close this one?

**Pull request closed without merging**